src/views/AuditDetail/AuditDetail.vue

```diff
@@ -172,7 +172,7 @@ import IconBaseDecorative from '../../components/icons/IconBaseDecorative';
 								audit.id,
 								this.sharecode,
 								(parameters) => {
-									this.parameters = parameters.filter(parameter => parameter.value && parameter.auditParameter.code != "SITE_SEEDS" && parameter.auditParameter.code != "PAGE_URLS" && parameter.auditParameter.code != "SCENARIO_ID" && parameter.auditParameter.code != "DOM_ID")
+									this.parameters = parameters.filter(parameter => parameter.value && parameter.auditParameter.code != "SITE_SEEDS" && parameter.auditParameter.code != "PAGE_URLS" && parameter.auditParameter.code != "SCENARIO_ID" && parameter.auditParameter.code != "DOM_ID" && parameter.auditParameter.code != "WEBDRIVER_BROWSER")
```

Comment:
Il serait peut être intéressant de créer une Map avec tous les parametres d'audit.
Il y a du code non fini juste apres : this.mainReference = 'to do'